### ISSUE TITLE
Linter doesn't report error in header files, just stops working.

### ISSUE THREAD

**Issue by XapaJIaMnu** [nirm03/SublimeLinter-clang#11]
It's really annoying when the linter stops working and you have to hunt down the issue. It's usually something not in the include path and can be discovered by running

```
clang -cc1 -fsyntax-only -fno-caret-diagnostics -fcxx-exceptions -Wall FILENAME.cpp
```

and then incrementally adding -I to accommodate the files that were not found. This error should be printed in the sublime window when such an include is added to at least show where the problem is and not have to hunt it down manually.

**Comment by dralley** [comment]
Problem mostly fixed by #14.  I'll see what can be done about a general case solution since it seems like the real problem is that certain errors (of which missing header is one type, but could be more) cause the linter to stop working without any further notification of that being the case.

Also see: https://github.com/nirm03/SublimeLinter-clang/issues/2

**Comment by XapaJIaMnu** [comment]
Hey,

I agree #14 would mostly fix it, but sometimes there is the problem that you have not set up the include directories correctly for your own project (especially if you are using a build system like cmake or bjam and a sufficiently large project). It's annoying to have to go to command line and type VERY long commands with the full include lists from your project just to see you have misspelled a directory name in your extra include paths.. It would be much better if the linter would just report this when it dies.

Cheers,

Nick
